index.html

```diff
@@ -1,58 +1,147 @@
+			flex-direction: row;
+			margin: 0 !important;
+			flex-basis: auto;
+		}
+		
+		#question {
+			width: 100px;
+			height: 30px;
+			color: white;
+			font-weight: lighter;
+			margin-bottom: 10px;
+			background-color: rgba(0, 0, 0, 0.6);
+			border-radius: 3px;
+			display: flex;
+			align-content: center;
+			justify-content: center;
+			align-items: center;
+		}
+		
+		#question:hover {
+			cursor: pointer;
+		}
+		
+		#infobox {
+			display: none;
+		}
+		
+		img {
+			width: 100%;
+			margin: 20px;
+		}
+		
+		h1 {
+			font-size: 3em;
+			color: white;
+			padding-top: 80px;
+			/*background-color: rgba(183, 58, 98, 0.89);*/
+		}
 	</style>
 </head>
 
 <body>
 	<!--<img style ="width: 80px; height: 80px; margin-top:10px" src ="./images/logo.png">-->
-    <h1 >The Coding Space Projects</h1>
-   <div id="container">
-	<div>
-		<a href="./scratch"><img style ="width:300px; height: 300px" id="banner1" src="./images/scratch2.png" onmouseover="this.src='./images/scratchhover.png'" onmouseout="this.src='./images/scratch2.png'" /></a>
-	</div>
-	<div>
-		<a href="http://woofjs.com/create"><img  style ="width:300px; height: 300px" id="banner3" src="./images/woof2.png" onmouseover="this.src='./images/woofhover.png'" onmouseout="this.src='./images/woof2.png'"/></a>
+	<h1>The Coding Space Projects</h1>
+	<div id="container">
+		<div>
+			<a href="./scratch"><img style="width:300px; height: 300px" id="banner1" src="./images/scratch2.png" onmouseover="this.src='./images/scratchhover.png'" onmouseout="this.src='./images/scratch2.png'" /></a>
+		</div>
+		<div>
+			<a href="http://woofjs.com/create"><img style="width:300px; height: 300px" id="banner3" src="./images/woof2.png" onmouseover="this.src='./images/woofhover.png'" onmouseout="this.src='./images/woof2.png'" /></a>
+		</div>
+		<div>
+			<a href="./web"><img style="width:300px; height: 300px" id="banner2" src="./images/web2.png" onmouseover="this.src='./images/webhover.png'" onmouseout="this.src='./images/web2.png'" /></a>
+		</div>
+		<div class="infoparts">
+			<div id="question">Learn More</div>
+			
+		</div>
+		<div id="infobox" class="infoparts">
+			<div class="textbox">Scratch is a block-based coding platform that lets you code games and animations without worrying about typing. All you need to do is drag and drop!</div>
+			<div class="textbox">WoofJS is a JavaScript framework that helps you transition from block-based coding in Scratch to syntax-based coding in JavaScript</div>
+			<div class="textbox">These Web projects use HTML, CSS, and JavaScript to create interactive webpages and games.</div>
+			<div class="moreinfo">These projects are all "un-tutorials." They are similar to tutorials in that they guide you in building something, except we don’t include any specific instructions about what code to use. You figure out the rest via tinkering, Googling, asking a friend, and iterating towards a solution, all while developing deep insights and intuition of high-level concepts. </div>
```

Comment:
I want to make this more student-friendly language. I also want to zoom out and explain how the entire curricular structure works as well, not just how the untutorials work. This is a challenging communication problem so I'd look towards other people who have similar structure that we admire, like The Recurse Center (recurse.com).

Some points:

1. "Untutorials" - I'm thinking a more understandable phrase might be better, like "Guided Projects" or "Project Outlines"
2. Self-paced, project-based, learn as you build
3. Student-directed - explain that you can explore these projects in whichever order you choose. We've organized the projects into a series of levels so you can find projects at your level.